### ISSUE TITLE
Bad request adding approaches at version 5.24.0

### ISSUE THREAD

**Issue by Rougree** [Project-OSRM/osrm-backend#5962]
Hi guys,

I'm doing the following request:

`'http://localhost:5000/route/v1/driving/polyline(mv%7E%7BH%7DgiZ%7DBbmA)?overview=false&steps=false&alternatives=false&geometries=polyline&annotations=false&continue_straight=default'`

This gives me normal good results! However, when I want to add `approaches=curb`. I get a bad request:

`'http://localhost:5000/route/v1/driving/polyline(mv%7E%7BH%7DgiZ%7DBbmA)?overview=false&steps=false&alternatives=false&geometries=polyline&annotations=false&continue_straight=default&approaches=curb'`

Does someone understand what the problem is?

**Comment by jcoupey** [comment]
For the second request, you should inspect the response even with a bad http status code as it contains valuable information, probably along the lines of:

```
{"message":"Number of elements in approaches size 1 does not match coordinate size 2","code":"InvalidOptions"}
```

As you can choose approaches per input coordinate, this parameter expects a list of values, see options in https://github.com/Project-OSRM/osrm-backend/blob/master/docs/http.md#requests.

**Comment by Rougree** [comment]
I see thank you! Is `approaches=curb` also available for table service? I'm using version 5.24.0. It is added to the general options right? If it is, I don't understand how you do a request with approaches=curb for table specific requests @jcoupey
